lukasz-lapczyk/review-dealer
# Add Facebook authentication

# Overview
Users can now sign up to the site using their Facebook accounts.

Environment variables (create .env file in root directory and paste the content below to the file; use .env.example as a reference):
FACEBOOK_APP_ID=170149593638351
FACEBOOK_SECRET=2a04f27fe9448dac5f885f4791102360

# Testing prerequisites
- bundle install

# Testing
- navigate to root directory
- click 'Sign up' link on navbar
- click 'Sign in with Facebook' link below the sign up form
- [x] you should be logged in with your Facebook account
